app/models/flood_risk_engine/organisation.rb

```diff
@@ -2,6 +2,7 @@ module FloodRiskEngine
   class Organisation < ActiveRecord::Base
     belongs_to :contact
     has_one :enrollment, dependent: :restrict_with_exception
+    has_many :partners
```

Comment:
Too late and irrelevant now in big scheme of things .. but not sure why STI was over kill, when its the correct abstraction ?  Coming from a C++/OO background it seemed very straight fwd to implement and understand. The enum adds complexity here with no benefit in managing behaviour.